docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+
+In Filecoin, users pay storage miners to store their files. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+
+![Diagram showing a four-step process for storing data with Filecoin. First step: User pays miner to store their data. Second step: Miner commits publicly via Filecoin’s blockchain to storing the data. Third step: The network constantly verifies that miners are storing data correctly. Fourth step: User pays miner to retrieve data.](./images/what-is-filecoin.png)
+
+## For users
+
+Filecoin provides users assurance their files have been stored safely and will be available in the future.
```

Comment:
Some folks on the Filecoin team have been telling me that it doesn't assure your files are stored safely, it only incentives that to happen and gives you the mechanism to prove it if they're not. @mishmosh how do you recommend we present this? 

```suggestion
Filecoin provides users assurance that their files have been stored safely and will be available in the future.
```

Reply:
Fair enough, assurance is too strong a word to use here. I have removed this claim and focused on the proof part instead,